includes/Core/Util/Migration_1_45_0.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/**
+ * Migration for 1.45.0
+ *
+ * @package   Google\Site_Kit\Core\Util
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Authentication\Authentication;
+use Google\Site_Kit\Core\Authentication\Google_Proxy;
+use Google\Site_Kit\Core\Authentication\Profile;
+use Google\Site_Kit\Core\Authentication\Verification_File;
+use Google\Site_Kit\Core\Authentication\Verification_Meta;
+use Google\Site_Kit\Core\Permissions\Permissions;
+use Google\Site_Kit\Core\Storage\Options;
+use Google\Site_Kit\Core\Storage\User_Options;
+use WP_User;
+use WP_Error;
+
+/**
+ * Class Migration_1_45_0
+ *
+ * @since 1.45.0
+ * @access private
+ * @ignore
+ */
+class Migration_1_45_0 {
+	/**
+	 * Target DB version.
+	 */
+	const DB_VERSION = '1.45.0';
+
+	const OPTION_KEY_PSI_UPDATED = 'googlesitekit_psi_updated';
+
+	/**
+	 * Context instance.
+	 *
+	 * @since 1.45.0
```

Comment:
All `@since` phpdoc tags in this file should be `n.e.x.t` as well as the `DB_VERSION` constant value.